src/extension.ts

```diff
@@ -286,6 +286,52 @@ export async function activate(context: vscode.ExtensionContext) {
 		),
 	);
 
+	context.subscriptions.push(
+		vscode.commands.registerCommand(
+			'intuita.executeAsPiranhaRule',
+			async (configurationUri: vscode.Uri) => {
+				const fileStat = await vscode.workspace.fs.stat(
+					configurationUri,
+				);
+				const configurationUriIsDirectory = Boolean(
+					fileStat.type & vscode.FileType.Directory,
+				);
+
+				if (!configurationUriIsDirectory) {
```

Comment:
do we need this check? this command is only available in right-click menu for directories, right?

Reply:
I would say better safe than sorry :) We can accidentally remove the restriction from `package.json` settings.